### PR TITLE
Override `isInQueue` & `getQueueItem` from `WorkflowJob`

### PR DESCRIPTION
**Pull request by jglick**
Just noticed that these have placeholder implementations (`false` / `null`) in `Job` which are overridden by `AbstractProject` (because it is a `Queue.Task`) and `WorkflowJob` should do the same.

### CODE REVIEW

**Review by julieheard — APPROVED** [review]

**Comment by jglick** [comment]
Need review from someone with write access: @olamy @rsandell @dwnusbaum @car-roll

**Review by dwnusbaum — APPROVED** [review]
Just curious, what is the expected impact of this change?

**Comment by jglick** [comment]
Good question. I searched for usages of these `Job` methods in core (excluding calls from inside `AbstractProject` itself) including Jelly and AFAICT it only affects the REST API (`@Exported`).

**Pull request merged**
